schema/ask_order_v1.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
hmmm not sure why these are here...

Reply:
auto-generated JSON schema generated by the Makefile.  It's nice to have because you can use quicktype.io to generate code (for example).

Reply:
gotcha i just thought it was odd cause i don't think i changed any of these